### PR TITLE
Update Samples

### PR DESCRIPTION
**Pull request by Css-IanM**
Added simple range examples for Horizontal and Vertical.

### CODE REVIEW

File: Source/Samples/Sections/Widgets/RangeSection.cs [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+using Gtk;
+
+namespace Samples
+{
+    [Section(ContentType = typeof(Range), Category = Category.Widgets)]
+    class RangeSection : ListSection
+    {
+        static HScale hScale;
+        static VScale vScale;
```
Comment (cra0zy) [inline Source/Samples/Sections/Widgets/RangeSection.cs]: No point in keeping these static.

File: Source/Samples/Sections/Widgets/RangeSection.cs [hunk 2]
```diff
@@ -0,0 +1,42 @@
+
```
Comment (cra0zy) [inline Source/Samples/Sections/Widgets/RangeSection.cs]: Redundant empty line.

File: Source/Samples/Sections/Widgets/RangeSection.cs [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+        public RangeSection()
+        {
+            AddItem(CreateHorizontalRange());
+            AddItem(CreateVerticalRange());
+        }
+
+        public (string, Widget) CreateHorizontalRange()
+        {
+            Adjustment adj;
+            adj = new Adjustment(0.0, 0.0, 101.0, 0.1, 1.0, 1.0);
```
Comment (cra0zy) [inline Source/Samples/Sections/Widgets/RangeSection.cs]: Use `var adj = `

File: Source/Samples/Sections/Widgets/RangeSection.cs [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+            hScale = new HScale(adj);
```
Comment (cra0zy) [inline Source/Samples/Sections/Widgets/RangeSection.cs]: Use `var hScale=`

File: Source/Samples/Sections/Widgets/RangeSection.cs (hunk 1 above)
Comment (Css-IanM) [reply Source/Samples/Sections/Widgets/RangeSection.cs]: cleaned up the reference from older docs to var syntax.

File: Source/Samples/Sections/Widgets/RangeSection.cs (hunk 3 above)
Comment (Css-IanM) [reply Source/Samples/Sections/Widgets/RangeSection.cs]: updated syntax

File: Source/Samples/Sections/Widgets/RangeSection.cs (hunk 2 above)
Comment (Css-IanM) [reply Source/Samples/Sections/Widgets/RangeSection.cs]: removed EOF empty line.

**Comment by cra0zy** [comment]
Thanks, merging!

**Pull request merged**